cloudhub-2/modules/ROOT/pages/ps-create-configure-tgw.adoc

```diff
@@ -1,7 +1,7 @@
 = Creating Transit Gateway Attachments
 
-Multiple private spaces can connect to an internal VPC.
-You can reuse the transit gateway attachment by connecting another private space to it and accepting the attachment. 
+When you attach a transit gateway to a private space, Amazon Web Services (AWS) creates a _transit gateway attachment_ to represent the connection. You can attach the same transit gateway to on-premisis VPCs in your own AWS account to enable communication between private spaces and VPCs. Multiple private spaces can connect to an internal VPC.
```

Comment:
Cloudhub 2.0 creates the attachment on Amazon Web Services (AWS)

Reply:
```suggestion
When you attach a transit gateway to a private space, CloudHub 2.0 creates a _transit gateway attachment_ to represent the connection on Amazon Web Services (AWS). You can attach the same transit gateway to on-premises VPCs in your own AWS account to enable communication between private spaces and VPCs. Multiple private spaces can connect to an internal VPC.
```